pymc/tests/test_distributions_moments.py

```diff
@@ -595,3 +596,18 @@ def test_discrete_uniform_moment(lower, upper, size, expected):
     with Model() as model:
         DiscreteUniform("x", lower=lower, upper=upper, size=size)
     assert_moment_is_expected(model, expected)
+
+
+@pytest.mark.parametrize(
+    "mu, cov, size, expected",
+    [
+        (np.array([1.]), np.array([[1.]]), None, np.array([1.])),
+        (np.ones((10, )), np.identity(10), None, np.ones((10, ))),
+        (np.ones(2), np.identity(2), 4, np.ones((4, 2))),
+        (np.ones(2), np.identity(2), (4, 2), np.ones((4, 2, 2))),
```

Comment:
I think it will be good to add another test when mu is 2-dimensional.

As an example -
`np.ones((2, 2)), np.identity(2), 4, np.ones((4, 2, 2))`

Reply:
I think mu cannot be 2-dimensional.
I am removing the test cases where mu is 2-dim.